datalad/ui/tests/test_base.py

```diff
@@ -85,3 +88,6 @@ def nothing(x, k=1):
     @with_testsui(responses='a')
     def ask():
         assert_equal(ui.question('what is a?'), 'a')
+
+
+    ask()
```

Comment:
AFAICS, this fix and the cosmetic changes above are unrelated to yesno default change, which isn't actually covered in the tests.

Reply:
> This one just spotted

That's fine, but it'd be easy to put that in a separate commit.  When reviewing, I find it confusing when unrelated bits are tossed in (including cosmetic fixups that just add noise to the diffs).